src/rdkafka_cgrp.c

```diff
@@ -95,6 +139,19 @@ rd_kafka_cgrp_max_poll_interval_check_tmr_cb (rd_kafka_timers_t *rkts,
 	 (rkcg)->rkcg_join_state ==				\
 	 RD_KAFKA_CGRP_JOIN_STATE_WAIT_REVOKE_REBALANCE_CB)
 
+/**
+ * @returns true if a rebalance is in progress.
```

Comment:
as noted, i'm going to replace this with an explicit boolean. this is not documented clearly because it's hard to think about, and subsequent testing confirmed my suspicion that i don't in fact know.